### PR TITLE
Feature networkproxymapping

### PR DESCRIPTION
**Pull request by pc0202**
Added Mapping APIs for Network hub service

### CODE REVIEW

File: apps/web-services/nodemon/nodemon-eagle-stage.json [hunk 1]
```diff
@@ -34,7 +36,7 @@
         "NODE_ENV": "development",
         "NODE_TLS_REJECT_UNAUTHORIZED": "0",
         "NOTIFICATIONS_API_BASE": "http://IP-ADDR:5805",
-        "PORTAL_PORT": "3003",
+        "PORTAL_PORT": "3004",
```
Comment (christyfernandes) [inline apps/web-services/nodemon/nodemon-eagle-stage.json]: Dont commit this change

File: apps/web-services/nodemon/nodemon-eagle-stage.json [hunk 2]
```diff
@@ -5,8 +5,9 @@
         "APP_LOGS": "/logs",
         "CASSANDRA_AUTH_ENABLED": false,
         "CASSANDRA_IP": "localhost",
-        "CASSANDRA_PASSWORD": "",
-        "CASSANDRA_USERNAME": "",
+        "CASSANDRA_PASSWORD": "cassandra",
```
Comment (christyfernandes) [inline apps/web-services/nodemon/nodemon-eagle-stage.json]: remove this change too

File: apps/web-services/nodemon/nodemon-eagle-stage.json [hunk 3]
```diff
@@ -5,8 +5,9 @@
...
+        "CASSANDRA_USERNAME": "cassandra",
```
Comment (christyfernandes) [inline apps/web-services/nodemon/nodemon-eagle-stage.json]: Remove this change too

File: apps/web-services/src/protectedApi_v8/network.ts [hunk 4]
```diff
@@ -10,15 +10,26 @@ const unknown = 'Network Apis:- Failed due to unknown reason'
 const apiEndpoints = {
   getConnectionEstablishedData: `${CONSTANTS.NETWORK_SERVICE_BACKEND}/connections/profile/fetch/established`,
   getConnectionRequestsData: `${CONSTANTS.NETWORK_SERVICE_BACKEND}/connections/profile/fetch/requested`,
+<<<<<<< HEAD
```
Comment (christyfernandes) [inline apps/web-services/src/protectedApi_v8/network.ts]: remove these conflicts

**Review by christyfernandes — CHANGES_REQUESTED** [review]

File: apps/web-services/nodemon/nodemon-eagle-stage.json (hunk 1 above)
Comment (karthik-tarento) [reply apps/web-services/nodemon/nodemon-eagle-stage.json]: Reverted back

File: apps/web-services/nodemon/nodemon-eagle-stage.json (hunk 3 above)
Comment (karthik-tarento) [reply apps/web-services/nodemon/nodemon-eagle-stage.json]: Reverted back

**Review by christyfernandes — APPROVED** [review]

**Pull request merged**